pennylane/gradients/pulse_gradient.py

```diff
@@ -466,7 +466,7 @@ def ansatz(params):
             times.append(process_time() - start)
 
     >>> print(times) # Show the gradient computation times in seconds.
-    [44.92675887899999, 10.405261402999997]
+    [55.75785480000002, 12.297400500000009]
```

Comment:
:sweat_smile:

Reply:
These numbers are dependent on the computer (and its CPU utilization) on which the code is executed, no?

Reply:
yes 💯 ... and the temperature of the computer and the room that computer is in 😄 and how many other things are running etc.